### PR TITLE
Adding logging utils to support local testing

### PR DESCRIPTION
**Pull request by stevewalton28**
In prep for exploratory testing, I wanted a way to quickly launch and relaunch a tail of all the docker logs for each container in a single multi pane tab in iTerm formatted as a grid so I can put it on a big screen and observe changes as I test.

I've scripted in osascript before for iTerm and didn't realise until I'd nearly finished that it's been deprecated and the iTerm team have moved to python.  If this proves popular with people then I'll take a minute to rewrite it in python.

Anyway - screenshot of the resulting iTerm window after the script has ran - as you can see each of the panes has the title of the container image that it represents (which works nicely for us because that's the name of the repo as well)

To us:

`chmod +x log-utils.sh`
`./log-utils.sh -l`

There's a second feature in the script to show us the disk usage for mounts, which will be helpful when testing as well.

`./log-utils.sh -d`

Screenshot 2022-08-11 at 22 08 33

### CODE REVIEW

**Review by SentientHousePlant — APPROVED** [review]
Lovely Steve, always good to create dev utilities :)

File: .env.example [hunk 1]
```diff
@@ -1,2 +0,0 @@
-# RENAME TO .env AND POPULATE
-GOV_NOTIFY_API_KEY=
```
Comment (srh-sloan) [inline .env.example:2]: Did you mean to delete the example env file?

**Review by srh-sloan — APPROVED** [review]
Nice little utility!

**Comment by gidsg** [comment]
@stevewalton28 you can commit the file permissions to git, so need to have this in the instructions.

e.g.
```
chmod a+x log-utils.sh
g commit -a
```

**Pull request closed without merging**